tests/test_suite.py

```diff
@@ -136,6 +136,14 @@ def parse_args():
                              dest="scenarios_regex",
                              default=None,
                              help="Regular expression matching test scenarios to run")
+    parser_rule.add_argument("--source", choices=['rule', 'profile'],
+                              default='rule',
+                              dest="target_source",
+                              help=("Specify wether target is a list of rules, or a profile. "
```

Comment:
@yuumasato Please change it to "whether".

Reply:
This doesn't apply anymore.